synapse/lib/aha.py

```diff
@@ -119,6 +197,11 @@ class AhaCell(s_cell.Cell):
             'type': ['string', 'array'],
             'items': {'type': 'string'},
         },
+        'provision:listen': {
+            'description': 'A telepath URL for the AHA provisioning listener.',
+            'type': ['string', 'array'],
```

Comment:
If an array is passed to dmon.listen() it will throw an exception when parsing the url parameter.

Reply:
Thanks! I was thinking of clients...